### PR TITLE
Nerfs the pandora box so that it dosen't insta teleports to anywhere it wants.

### PR DESCRIPTION
**Pull request by Moccha-Bee**
## About The Pull Request
Yes i am salty deal with it lol.

## Why It's Good For The Game
Pandora boxes could just keep teleporting and avoiding fights until the miner died from either it or the ash storm.

## Changelog
:cl:
balance: nerfed insta teleport from pandora.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
@ike709 can we add a "i ded" label?

**Comment by Moccha-Bee** [comment]
> @ike709 can we add a "i ded" label?

the pandora box was supposed to have a delay but whoemver did it fucked it up and in the end there is no cool down.

**Comment by ghost** [comment]
> > @ike709 can we add a "i ded" label?
> 
> the pandora box was supposed to have a delay but whoemver did it fucked it up and in the end there is no cool down.

it has a delay, you cant instantly teleport away right after teleporting.

**Comment by Moccha-Bee** [comment]
> > > @ike709 can we add a "i ded" label?
> > 
> > 
> > the pandora box was supposed to have a delay but whoemver did it fucked it up and in the end there is no cool down.
> 
> it has a delay, you cant instantly teleport away right after teleporting.

a femto second delay lol

**Comment by Sapfearon** [comment]
You know, while he maybe salty there is something to it.
There need to be a system to FORCE spawned monster to engage miner in fight instead of running around like chicken or god forbid, hiding inside lava.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/pandora.dm [hunk 1]
```diff
@@ -125,7 +125,7 @@
 			new /obj/effect/temp_visual/hierophant/blast/pandora(t, src)
 
 /mob/living/simple_animal/hostile/asteroid/elite/pandora/proc/pandora_teleport(target)
-	ranged_cooldown = world.time + cooldown_time
+	ranged_cooldown = world.time + (cooldown_time * 3)
```
Comment (axpov) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/pandora.dm:128]: Not where you should be altering cooldown time. Just change it from 20 to 60 where it's defined

**Review by axpov — APPROVED** [review]

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/pandora.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/pandora.dm:128]: That would affect the cooldown of all attacks

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**